google/async_resumable_media/requests/download.py

```diff
@@ -14,14 +14,23 @@
 
 """Support for downloading media from Google APIs."""
 
+import logging
+
 import urllib3.response
 
-from google.resumable_media import _download
-from google.resumable_media import common
-from google.resumable_media import _helpers
-from google.resumable_media.requests import _request_helpers
+from google.async_resumable_media import _download
+from google.async_resumable_media import common
+from google.async_resumable_media import _helpers
+from google.async_resumable_media.requests import _request_helpers
+from google.resumable_media import _helpers as sync_helpers
 
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Add comment about why this is necessary.

Reply:
Removed this duplication.